README.md

```diff
@@ -97,3 +98,43 @@ since the commit hash is used as the image tag.
    ```bash
       hubploy deploy <hub-name> hub prod
    ```
+
+## How to use an unreleased helm chart
+
+Sometimes you might need to use the latest version of a helm chart and that isn't yet released.
+The following steps shows how to do so for the DaskHub project, but the pattern
+should be replicable for others.
+
+1. Clone the repo holding the latest version of the helm-chart and checkout the wanted branch:
+
+   *DaskHub example:*
+
+   ```bash
+      git clone https://github.com/dask/helm-chart.git
+      git checkout master
+   ```
+2. Inside the `yaml` file holding the dependencies, add the local path to the helm-chart of the project
+you just checked out and use the version inside the `chart.yaml` or run `chartpress` to generate one if
+that is not available:
+
+   *DaskHub example:*
+
+   ```bash
+      dependencies:
+       - name: daskhub
+         repository: file://<path-to-your-daskhub-checkout>/daskhub
+         version: <version-from-chart.yaml-of-daskhub>
+   ```
+
+3. Deploy the changes.
+
+   **Note** If after deploying these changes some of the services don't show up and they are part of projects that
```

Comment:
I think you should run `helm dep up` regardless of what happens - I think otherwise this *will* happen.

Reply:
Or rather, this should be run any time the dependencies (in requirements.yaml or Chart.yaml) of the chart you just cloned changes.

Reply:
:+1: 

- `helm repo update`: fetching the latest information from a Helm chart repository's index.yaml file which lists all available versions and charts in the Helm chart repository. Not having done this can make a `helm upgrade jupyterhub/jupyterhub --version 1.2.3` fail because one doesn't have information about the new published version locally yet even though it is available.
- `helm dep update`: this will use available chart information to update a local copy put in the /charts folder within the Helm chart, it is a folder where local copies of Helm chart reside.

If you use a Helm chart dependency being `file://`, then you won't need `helm repo update` for that dependency at least, but you always need `helm dep up` I think to copy the dependency to the `/charts` folder alongside `/templates`

Reply:
Thank you for all the useful info :heart: